e2e/next-sandbox/utils/Button.tsx

```diff
@@ -30,6 +31,7 @@ export default function Button(props: Props) {
         opacity,
       }}
       title={`#${props.id}`}
+      disabled={!enabled}
```

Comment:
This feels a bit weird from the outside but I can see the readability benefits of doing `enabled={canSet}`.